### PR TITLE
Custom Religious Icons

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Added a lot of new religious icons (counting both recoloured old and new ones)
- Sorted which vanilla icons can be used when reforming mod religions

#### How to test:
- Check available icons when creating a new faith

### CODE REVIEW

**Comment by TheDrPibb1999** [comment]
There is a blank icon space right here 
Capture

File: common/religion/religions/ape_group.txt [hunk 1]
```diff
@@ -57,7 +57,42 @@
...
+		# Leaf
+		baltic_custom
+		# Vine
+		custom_faith_3
+		# Tree
+		custom_faith_6 waaqism custom_faith_fp1_yggdrasil
+		# Flower
+		custom_faith_10
+		# Elephant
+		custom_faith_5
```
Comment (zumbak04) [inline common/religion/religions/ape_group.txt:73]: There are no elephants on Azeroth. Should be removed from other religions too... except the draenei maybe.

File: common/religion/religions/ape_group.txt [hunk 2]
```diff
@@ -57,7 +57,42 @@
...
+		# Horse
+		custom_faith_7
```
Comment (zumbak04) [inline common/religion/religions/ape_group.txt:75]: No horses near hozens.

File: common/religion/religions/ape_group.txt [hunk 3]
```diff
@@ -57,7 +57,42 @@
...
+		# Cat
+		custom_faith_9
+		# Deer
+		finno_ugric_custom
```
Comment (zumbak04) [inline common/religion/religions/ape_group.txt:79]: No deers near hozens.

File: common/religion/religions/ape_group.txt [hunk 4]
```diff
@@ -57,7 +57,42 @@
...
+		# Stork
+		melieism
+		# Raven
+		germanic_custom custom_faith_fp1_odins_ravens
+		# Dog
+		kushism custom_faith_fp1_fenrir
```
Comment (zumbak04) [inline common/religion/religions/ape_group.txt:85]: No dogs near hozens.

File: common/religion/religions/daradorism.txt [hunk 5]
```diff
@@ -57,7 +57,44 @@
...
+		custom_faith_6 waaqism custom_faith_fp1_yggdrasil
+		# Flower
+		custom_faith_10
+		# Fauna
+		# Elephant
+		custom_faith_5
+		# Horse
+		custom_faith_7
+		# Cat
+		custom_faith_9
```
Comment (zumbak04) [inline common/religion/religions/daradorism.txt:79]: Don't think there is a lot of cats in Northrend.

File: common/religion/religions/daradorism.txt [hunk 6]
```diff
@@ -57,7 +57,44 @@
...
+		# Deer
+		finno_ugric_custom
+		# Stork
+		melieism
```
Comment (zumbak04) [inline common/religion/religions/daradorism.txt:83]: Don't think there is a lot of cats in Northrend.

File: common/religion/religions/ape_group.txt (hunk 2 above)
Comment (MemeWizard) [reply common/religion/religions/ape_group.txt:75]: Done

File: common/religion/religions/ape_group.txt (hunk 3 above)
Comment (MemeWizard) [reply common/religion/religions/ape_group.txt:79]: Done

File: common/religion/religions/ape_group.txt (hunk 4 above)
Comment (MemeWizard) [reply common/religion/religions/ape_group.txt:85]: Done

File: common/religion/religions/daradorism.txt (hunk 5 above)
Comment (MemeWizard) [reply common/religion/religions/daradorism.txt:79]: These religion are present in both Northrend and EK

File: common/religion/religions/daradorism.txt (hunk 6 above)
Comment (MemeWizard) [reply common/religion/religions/daradorism.txt:83]: These religion are present in both Northrend and EK

File: common/religion/religions/ape_group.txt (hunk 1 above)
Comment (MemeWizard) [reply common/religion/religions/ape_group.txt:73]: Removed from religions except draenei (elekks), daradorism and shamanism (Behemoth and Mammoths)

https://wowpedia.fandom.com/wiki/Behemoth
https://wowpedia.fandom.com/wiki/Mammoth

File: common/religion/religions/lizard_group.txt [hunk 7]
```diff
@@ -57,7 +57,47 @@
...
+		# Stork
+		melieism
+		# Raven
+		germanic_custom custom_faith_fp1_odins_ravens
+		# Dog
+		kushism custom_faith_fp1_fenrir
+		# Goat
+		pagan_magyar
+		# Bear
+		siberian
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/religion/religions/lizard_group.txt:86]: Done

File: common/religion/religions/pandaren_religion_group.txt [hunk 8]
```diff
@@ -57,7 +57,48 @@
...
+		# Stork
+		melieism
+		# Raven
+		germanic_custom custom_faith_fp1_odins_ravens
+		# Dog
+		kushism custom_faith_fp1_fenrir
+		# Goat
+		pagan_magyar
+		# Bear
+		siberian
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/religion/religions/pandaren_religion_group.txt:95]: Done

File: common/religion/religions/titanic_group.txt [hunk 9]
```diff
@@ -57,7 +57,34 @@
 	}
 
 	custom_faith_icons = {
-		custom_faith_1 custom_faith_2 custom_faith_3 custom_faith_4 custom_faith_5 custom_faith_6 custom_faith_7 custom_faith_8 custom_faith_9 custom_faith_10 dualism_custom_1 zoroastrian_custom_1 zoroastrian_custom_2 buddhism_custom_1 buddhism_custom_2 buddhism_custom_3 buddhism_custom_4 taoism_custom_1 yazidi_custom_1 sunni_custom_2 sunni_custom_3 sunni_custom_4 ibadi_custom muhakkima_1 muhakkima_2 muhakkima_4 muhakkima_5 muhakkima_6 judaism_custom_1 custom_faith_fp1_fenrir custom_faith_fp1_irminsul custom_faith_fp1_jormungandr custom_faith_fp1_odins_ravens custom_faith_fp1_runestone_moon custom_faith_fp1_thors_hammer custom_faith_fp1_valknut custom_faith_fp1_yggdrasil
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/religion/religions/titanic_group.txt:60]: Added more

File: common/religion/religions/ape_group.txt [hunk 10]
```diff
@@ -57,7 +57,34 @@
...
+		# Leaf
+		baltic_custom
+		# Vine
+		custom_faith_3
+		# Tree
+		custom_faith_6 waaqism custom_faith_fp1_yggdrasil
+		# Flower
+		custom_faith_10
+		# Cat
+		custom_faith_9
```
Comment (zumbak04) [inline common/religion/religions/ape_group.txt:73]: No small cats near hozens.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/religions/ape_group.txt [hunk 11]
```diff
@@ -57,7 +57,34 @@
...
+		# Stork
```
Comment (MemeWizard) [inline common/religion/religions/ape_group.txt:74]:
```suggestion
		custom_faith_10
		# Stork
```

File: common/religion/religions/ape_group.txt (hunk 10 above)
Comment (MemeWizard) [reply common/religion/religions/ape_group.txt]: Done

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by TheGroxEmpire — COMMENTED** [review]
Religion group belonging these religions are lacking in more varied option IMO:
<details>

image
image
image
image
image
</details>

**Comment by MemeWizard** [comment]
@TheGroxEmpire, added a bunch of new/recoloured icon for nerubian and elemental faiths.

Didn't add more for void fellas because I have no idea what to give them.

**Review by TheGroxEmpire — COMMENTED** [review]
> Didn't add more for void fellas because I have no idea what should I give them.

I think the void could use a void version of the Naaric like dark Naaru
<details>

image
</details>

Which reminds me:

- The Naaric could use some Naaru icon.
<details>

image
</details>

**Comment by MemeWizard** [comment]
> The Naaric could use some Naaru icon.

Added a bunch

> void

Added a bit

**Review by TheGroxEmpire — APPROVED** [review]
My suggestions were implemented.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by TheDrPibb1999 — APPROVED** [review]
All looks great to me!! all things implemented imo were for the better

**Pull request merged**